Dockerfile.debug

```diff
@@ -1,6 +1,6 @@
 #### Please note that this dockerfile is not used to release the image,
 #### It is used merely to create a test image to run tests or to be uploaded manually for testing pourpuses
-FROM golang:1.20-alpine AS build
+FROM golang:1.20.6-bookworm AS build
```

Comment:
🤔 should we use `golang:1.20.6-alpine` instead?

Reply:
It is just a test image, therefore since there is no risk I prefer to unify.

Or you were interested in the image size?

Reply:
👍 Unifying is fine with me. I asked just in case it made sense to keep it _as it was_.